app/views/users/show.html.erb

```diff
@@ -1,2 +1,5 @@
 <h1><%= t(".welcome") %></h1>
-<p><%= t(".profile_information", email: user.email) %></p>
+<ul>
+  <li><%= t(".name") %> :<%= current_user.name %></li>
```

Comment:
The previous comment will dry this view as well.

Reply:
Thanks Elle!